unity/Assets/0Refactored/scenes/experiments/HuygensPrinciple/HuygensPrinciple.pc.unity

```diff
@@ -592,612 +592,57 @@ PrefabInstance:
     - target: {fileID: 1868487302547840112, guid: 6a4ebf0959b5c954e90ab6b8622ee677,
```

Comment:
The HuygensPrinciple.pc scene contains the InterferencePlate object but it is not applied to the HuygensExperiment prefab. Please add the InterferencePlate prefab to the HuygensExperiment prefab so that the interference plate is automatically available in all HuygensPrinciple experiment scenes.